### ISSUE TITLE
Helmet 6.1.2: 'typeerror (0 helmet_1.default) is not a function' when using helmet 

### ISSUE THREAD

**Issue by ctanejaex2** [helmetjs/helmet#415]
```
import helmet from 'helmet';
app.use(helmet())
```

Using helmet gives error **typeerror (0 helmet_1.default) is not a function** on `app.use(helmet)`. 
This was working fine util day before yesterday. Any solutions? :/

**Comment by EvanHahn** [comment]
In the short term, you can fix this by installing `helmet@6.0.1`.

What does your `tsconfig.json` file look like?

**Comment by ctanejaex2** [comment]
I will try installing helmet6.0.1
Furthermore, here is my tsconfig.json 
````{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "es2017",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "skipLibCheck": true,
    "strictNullChecks": false,
    "noImplicitAny": false,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false
  }
}
```

**Comment by EvanHahn** [comment]
I'm unable to reproduce your issue. Did you specify `"type": "module"` in your `package.json`? Could you create a small sample project?

**Comment by EvanHahn** [comment]
@actraiser I can't reproduce this issue. Could you create a sample project or show me your `tsconfig.json`?

**Comment by actraiser** [comment]
Here is a repo with a freshly initialized nestjs application that uses helmet in src/main.ts 

on yarn run start it will the throw the error when using 6.1.2 - otherwise (with 6.0.1) no error is thrown.

Greets
-act

**Comment by EvanHahn** [comment]
@actraiser Thank you for the sample app. I can reproduce it and will prepare a fix ASAP.

For anyone experiencing this problem, `helmet@6.0.1` should work fine until I fix it.

**Comment by EvanHahn** [comment]
@actraiser Strangely, `yarn build` works fine but `yarn start` doesn't. I don't know why but am investigating a fix.

**Comment by EvanHahn** [comment]
This should be fixed in `helmet@6.1.3`. Let me know if you're still running into problems!
